test/e2e/statefulset_arbitrary_config/statefulset_arbitrary_config_test.go

```diff
@@ -22,9 +22,11 @@ func TestMain(m *testing.M) {
 }
 
 func TestStatefulSetArbitraryConfig(t *testing.T) {
+	// creates operator/deployment from the yaml files
 	ctx := setup.Setup(t)
 	defer ctx.Teardown()
 
+	// creates basic mongodb resource/custom resource
```

Comment:
I think it's fairly clear here what the method `NewTestMongoDB` does -- so I think the comment is redundant. we can remove it.

Reply:
leftovers of my own debugging process :)